src/util.ts

```diff
@@ -37,7 +37,11 @@ export const checkAndSanitizePath = (pathOrUrl: string): string => {
     const isUrlRegEX =
         /(https?:\/\/)([\w\-])+\.{1}([a-zA-Z]{2,63})([\/\w-]*)*\/?\??([^#\n\r]*)?#?([^\n\r]*)/g;
     if (isUrlRegEX.test(pathOrUrl)) {
-        return pathOrUrl;
+        let url = pathOrUrl;
+        if (url.includes("dropbox")) {
+            url = url.replace("www.dropbox.com", "dl.dropboxusercontent.com");
```

Comment:
Would it make sense to just replace `dropbox.com` and leave out the `www.`? That seems like an optional part of the link and could be missing in some circumstances